doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -280,6 +280,20 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-4-1 v2-4-0
```

Comment:
should v2.4.2 be added here as well?

Reply:
right, @ArekBalysNordic: Does this known issue (KRKNWK-17535)  need update too?